lib/fieldhand/paginator.rb

```diff
@@ -74,10 +78,10 @@ def items(verb, parser_class, query = {})
 
     private
 
-    def parse_response(query = {})
-      response = request(query)
-      raise ResponseError, response unless response.is_a?(::Net::HTTPSuccess)
+    attr_writer :retries
```

Comment:
Having a private `attr_writer` will raise a warning in earlier versions of Ruby, if you want to truly make it private, we can do it like this instead:

```ruby
class Paginator
  # ...
  attr_writer :retries
  private :retries
```